modules/dop/services/issue/export.go

```diff
@@ -34,6 +38,92 @@ type issueStage struct {
 	Value string
 }
 
+func (svc *Issue) Export(req *apistructs.IssueExportExcelRequest) (uint64, error) {
+	req.PageNo = 1
+	req.PageSize = 99999
+	fileReq := apistructs.TestFileRecordRequest{
+		ProjectID:    req.ProjectID,
+		OrgID:        uint64(req.OrgID),
+		Type:         apistructs.FileIssueActionTypeExport,
+		State:        apistructs.FileRecordStatePending,
+		IdentityInfo: req.IdentityInfo,
+		Extra: apistructs.TestFileExtra{
+			IssueFileExtraInfo: &apistructs.IssueFileExtraInfo{
+				ExportRequest: req,
```

Comment:
pageSize just for record or really used?

Reply:
really used